ios/sound.h

```diff
@@ -1,8 +1,8 @@
 /******************************************************************************\
- * Copyright (c) 2004-2020
+ * Copyright (c) 2004-2021
  *
  * Author(s):
- *  ann0see based on code from Volker Fischer
+ *  ann0see and ngocdh based on code from Volker Fischer
```

Comment:
In fact, you could remove me here since you did most of the work:

```suggestion
 *  ngocdh based on code from Volker Fischer
```

Reply:
I'll keep credit where credit is due :)